<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>202</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_BaseUrl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Server URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="kcfg_BaseUrl">
        <property name="toolTip">
         <string>The URL of the Open-Xchange server, should be something like https://myserver.org/</string>
        </property>
        <property name="whatsThis">
         <string>The URL of the Open-Xchange server, should be something like https://myserver.org/</string>
        </property>
        <property name="placeholderText">
         <string notr="true">https://myserver.org</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_Username">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="kcfg_Username">
        <property name="toolTip">
         <string>The username that is used to log into the Open-Xchange server</string>
        </property>
        <property name="whatsThis">
         <string>The username that is used to log into the Open-Xchange server</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_Password">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="kcfg_Password">
        <property name="toolTip">
         <string>The password that is used to log into the Open-Xchange server</string>
        </property>
        <property name="whatsThis">
         <string>The password that is used to log into the Open-Xchange server</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="checkConnectionButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Test Connection...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
